bin/data-push.js

```diff
@@ -74,6 +74,12 @@ Promise.resolve().then(async () => {
     } else {
       dataset = await prepareDatasetFromFile(filePath)
     }
+    
+    dataset._resources.forEach(resource => {
+      if (resource.constructor.name === 'FileInline')
```

Comment:
@Mikanebu not sure if this syntax is correct - can you use if statement without a block `{}`?

Reply:
@anuveyatsu yes, we can, but it is highly recommended that we always use braces, so someone can expand it in the future. Let me fix it quickly